README.md

```diff
@@ -0,0 +1,57 @@
+# Backend
+
+## Using the backend for the first time
+
+After cloning the backend to your local repository you need to perform the following steps, in order to start start the flask application.
+
+<ol>
+    <li>
+        Create a python virtual environment (venv) in your backend directory:
+        <ul>
+            <li>
+                (terminal): python3 -m venv .venv or python -m venv .venv
+            </li>
+        </ul>
+    </li>
+    <li>
+        Install all necessary dependencies with the provided requirements.txt file:
+        <ol>
+            <li>
+                Change into the newly created virtual environment:
+                <ul>
+                    <li>
+                        (terminal): cd .venv
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Activate the enviroment
+                <ul>
+                    <li>
+                        (terminal): source bin/activate
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Install the required dependencies:
+                <ul>
+                    <li>
+                        (terminal): pip install -r ../requirements.txt
```

Comment:
Again, just a minor addition.
Windows users need to use a backslash: `pip install -r ..\requirements.txt`. Would be nice if you could also add this.

Reply:
I added the install command with backslashes